src/OrbitGl/App.cpp

```diff
@@ -338,6 +339,19 @@ void OrbitApp::OnCaptureStarted(const orbit_grpc_protos::CaptureStarted& capture
         main_window_->AppendToCaptureLog(MainWindowInterface::CaptureLogSeverity::kInfo,
                                          absl::ZeroDuration(), "Capture started.");
 
+        orbit_version::Version capture_version{capture_started.orbit_version_major(),
+                                               capture_started.orbit_version_minor()};
+        orbit_version::Version current_version = orbit_version::GetVersionNumber();
+        if (capture_version > current_version) {
+          std::string warning_message = absl::StrFormat(
+              "The capture was taken with Orbit version %d.%d, which is higher than the "
+              "current version. Please open the capture using Orbit v%d.%d or above.",
+              capture_version.major_version, capture_version.minor_version,
+              capture_version.major_version, capture_version.minor_version);
```

Comment:
Pers pref: use `absl::Substitute` instead of `StrFormat` here.

Reply:
done.